src/components/SelectedItemBadge/SelectedItemBadge.tsx

```diff
@@ -37,28 +43,24 @@ type PriorityProps<PriorityItemValue> = {
     buttonLabel: string;
 };
 
-export interface Props<PriorityItemValue, ChildrenItemValue>
+export interface Props<PriorityItemValue>
     extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'> {
     /** Children nodes to be rendered within the Dropdown,
      *  which is triggered by the main button
      * */
     children?:
         | React.ReactElement
-        | React.ReactElement<ListItemProps>
-        | (React.ReactElement<ListItemProps> | EmptyElement)[]
+        | React.ReactElement<SingleSelectItemProps>
```

Comment:
Maybe we could simplify the children type? before it was so complicated due to the required type in `Dropdown`, now it looks much simpler there. Also, will `SingleSelectItemProps` cover all other options we might pass?